src/LDLFactorizations.jl

```diff
@@ -192,6 +199,13 @@ function ldl_numeric_upper!(n, Ap, Ai, Ax, Cp, Ci, Lp, parent, Lnz, Li, Lx, D, Y
       Lnz[i] += 1
       top += 1
     end
+    if dynamic_regul && abs(D[k]) < eps(T) * dynamic_dict[:Amax]
```

Comment:
If we ask the user to give us `Amax`, we might as well simply ask for a tolerance `tol`. The user can set it to `eps(T) * Amax`. What do you think?

Reply:
Yes good idea!